src/packets.c

```diff
@@ -163,6 +163,26 @@ TbBool process_dungeon_control_packet_spell_overcharge(long plyr_idx)
     struct Dungeon* dungeon = get_players_dungeon(player);
     SYNCDBG(6,"Starting for player %d state %s",(int)plyr_idx,player_state_code_name(player->work_state));
     struct Packet* pckt = get_packet_direct(player->packet_num);
+
+    while (game.conf.rules.magic.max_level_by_hold_shift_key && key_modifiers == KMod_SHIFT)
```

Comment:
It's hardcoded to shift now. Probably a good idea to for example link it to the possession gamekey. Check out `enum GameKeys`.